File: 프로그래머스/이상한 문자 만들기.java
class Solution {
    public String solution(String s) {
        String answer = "";
        
        String[] arr = s.split("");
        int cnt = 0;
        
        for(int i=0; i<s.length(); i++){
            cnt = (arr[i]==" ") ? 0 : cnt+1;
            answer += cnt%2== 0 ? arr[i].toLowerCase() : arr[i].toUpperCase();
        }
        
        return answer;
    }
}

File: 프로그래머스/Level1/이상한 문자 만들기.js
function solution(s) {
    let answer = s.split(' ')
    
    for(let i=0; i<answer.length; i++)
        answer[i] = answer[i].split('').map((elem, index) => index%2 === 0 ? elem.toUpperCase() : elem.toLowerCase()).join('');
    
    return answer.join(' ');
}
